### ISSUE TITLE
Create accessibility tests for all default framework widgets

### ISSUE THREAD

**Issue by CaseyHillers** [flutter/flutter#115318]
The entire Flutter SDK's widget catalog should be run against an accessibility test suite to verify they meet the minimum guidelines. Currently, there are Google tests that are run very late in the CI lifecycle that detect these breakages, but it would be good to catch it on GitHub.

For example, a select all semantics node was added in a widget, but it was too small to meet Android's accessibility guidelines for tappable nodes in https://github.com/flutter/flutter/pull/107193

Googlers, see b/258829761 for internal context, and references to the existing tests. We may able to port some of these externally to benefit the community.

**Comment by CaseyHillers** [comment]
\cc @justinmc
